### DISCUSSION TITLE
Cannot set the Tier in `RestoreObject`

### DISCUSSION THREAD

**Discussion by wunderbarb** [aws/aws-sdk-go-v2#2016] [Q&A]
The operation succeeds with the following code
```golang
	roIN := &awsS3.RestoreObjectInput{
		Bucket: aws.String(bucket),
		Key:    aws.String(key),
		RestoreRequest: &types.RestoreRequest{
			Days: days,
			// Description: aws.String("restore " + path),
			// GlacierJobParameters: &types.GlacierJobParameters{
			// 	Tier: "Bulk",
			// },
			// Tier: "Bulk",
			// Type: "SELECT",
		},
	}
	_, err = c.RestoreObject(ctx, roIN)
```
But when defining the tier as follows, it fails with an error `api error MalformedXML: The XML you provided was not well-formed or did not validate against our published schema`
```golang
	roIN := &awsS3.RestoreObjectInput{
		Bucket: aws.String(bucket),
		Key:    aws.String(key), 
		RestoreRequest: &types.RestoreRequest{
			Days: days,
			Tier: "Bulk",
			// Type: "SELECT",
		},
	}
	_, err = c.RestoreObject(ctx, roIN)
```
What am I doing wrong?  Thanks in advance

**Comment by RanVaknin** [comment]
Hi @wunderbarb 

I have to admit I spent a bit of time on this. S3's API is quite convoluted. My best guess of why you are seeing this validation errors, is that the S3 API evolves over time. Once glacier was introduced it was added on as an argument to that generic request, and that argument has its own tier.

Are you not targeting any glacier specific storage tier?
